guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,600 @@
+
+Next we will transform the generated Dockerfile `ENTRYPOINT` instruction:
+
+```
+lazy val myApp = project("my-app")
+  .settings(
+    // Add the following line within the module settings
+    dockerCommands :=
+      dockerCommands.value.flatMap {
+        case ExecCmd("ENTRYPOINT", args @ _*) => Seq(Cmd("ENTRYPOINT", args.mkString(" ")))
+        case v => Seq(v)
+      }
+   )
+```
+
+As part of the Docker build, SBT Native Packager will also generate a startup script for the application. This startup script is referenced by the `ENTRYPOINT` command in the Dockerfile, i.e. `ENTRYPOINT ["bin/myapp"]`. Note the value of the `ENTRYPOINT` is an array - this is the `exec` form declaration of the `ENTRYPOINT`.
+
+The script above transforms `ENTRYPOINT` from `exec` form to `shell` form. In the `shell` form, the `ENTRYPOINT` is declared as a simple string value. This string value will be executed by the container's shell using `sh -c`. The `shell` form is required to ensure the environment variables declared in the `dockerEntrypoint` argument is sourced from the shell within the Docker container. Refer to Docker [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#entrypoint) documentation for the difference between `exec` and `shell` form.
+
+The container image will be published to the the default Docker repository specified by SBT Native Packager. To publish to a different repository, set the `dockerRepository` setting to the repository you wish to publish. Please note this is an optional step. The Kubernetes deployment in this guide expects the image to be located at `mygroup/myapp`. To match the repository we need to set the `dockerRepository` to `mygroup`:
+
+```
+lazy val myApp = project("my-app")
+  .settings(
+    // Add the following line within the module settings
+    dockerRepository := Some("mygroup")
+  )
+```
+
+When the image is published, the version tag will be derived from the SBT project version. However it's possible to also publish both the `latest` tag and the version tag by enabling the `dockerUpdateLatest` setting. Please note this is an optional step. The Kubernetes deployment in this guide expects the `latest` image tag to be present. To match this expectation we need to enable `dockerUpdateLatest` setting:
+
+```
+lazy val myApp = project("my-app")
+  .settings(
+    // Add the following line within the module settings
+    dockerUpdateLatest := true
+  )
+```
+
+Execute the following command to containerize and publish your application's Docker image:
+
+```bash
+sbt docker:publishLocal
+```
+
+Additional SBT setting documentation to control the Docker image build process is available at the [Docker Plugin](http://www.scala-sbt.org/sbt-native-packager/formats/docker.html?highlight=dockercommand) documentation page from SBT Native Packager.
```

Comment:
The SBT Native Packager [Docker Plugin](http://www.scala-sbt.org/sbt-native-packager/formats/docker.html?highlight=dockercommand) documentation provides more information on controlling the Docker image build process.